bin/load_phenotypes_spreadsheet_csv.pl

```diff
@@ -167,6 +168,11 @@ =head1 AUTHOR
 }
 
 my $store_phenotypes = CXGN::Phenotypes::StorePhenotypes->new(
+    basepath=>$opt_b,
+    dbhost=>$opt_H,
```

Comment:
This should not be part of the object, but done outside of the object and used to create a schema???

Reply:
The reason I added those params is that now during phenotype deletion it runs:
my $async_delete = CXGN::Tools::Run->new();
$async_delete->run_async("perl $basepath/bin/delete_nd_experiment_entries.pl -H $dbhost -D $dbname -U $dbuser -P $dbpass -i $nd_experiment_ids");

The PhenotypeStore runs phenotype deletion in the case of overwriting previously loaded phenotype values, and so I needed those params in order to call the async_delete in the object.

Is there a better way?

Reply:
I guess the nd_experiment entries is the slow part to delete? That's why you factored it out to a script?

Reply:
the main thing this PR does is factor out deletion of nd_experiment entries into an asynchronous process because it is extremely slow as the database become large

Reply:
Not sure if there is a better way...  let me sleep on it..

Another thing is that I completely re-factored CXGN::Trial in another branch, so we'll have to be careful when I merge that one...